docs/starters.yml

```diff
@@ -796,3 +796,23 @@
     - Based on default starter
     - Bulma Css
     - Sass based Styling
+- url: https://gatsby-starter-mate.netlify.com
+  repo: https://github.com/EmaSuriano/gatsby-starter-mate
+  description: A portfolio starter for Gatsby integrated with Contentful CMS.
+  tags:
+    - Styled-components
+    - Contentful
+    - Portfolio
+  features:
+    - Gatsby v2
+    - Rebass: styled component system
```

Comment:
I think the `:` will make (Linting) problems. Better replace that with something else?